gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed Mar 29 15:48:21 CEST 2023
+#Mon May 20 12:47:38 CEST 2024
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
 distributionPath=wrapper/dists
-zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
why not latest?

Reply:
Bumping higher requires more changes and extends the scope. I would prefer to not extend the scope too much since we're already introducing a lot of changes